c_binding/libata.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2016 Red Hat, Inc.
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
```

Comment:
@tasleson No worries, hope you had a good break! I agree with keeping everything within libstoragemgmt as LGPL 2.1, and appreciate you tracking down that compatibility FAQ. I don't think that @cathay4t's code would execute as a totally separate process from ALL libstoragemgmt code, would it? So should he change that to LGPL 2.1 too?